engine/classes/Elgg/Database/MetadataTable.php

```diff
@@ -123,6 +125,10 @@ function create($entity_guid, $name, $value, $value_type = '', $owner_guid = 0,
 		}
 	
 		$access_id = (int) $access_id;
+		
+		if (strlen($value) > self::MYSQL_TEXT_BYTE_LIMIT) {
+ 			elgg_log("Metadata {$name} is above the MySQL TEXT size limit and may be truncated.", 'WARNING');
```

Comment:
This line and a few others have a space at the beginning of the line. Remove them.